motor/discovery/aws/ecr.go

```diff
@@ -140,6 +128,134 @@ func (a *EcrImages) getRepositories() []*jobpool.Job {
 
 // combine with docker image MondooContainerImageID
 func MondooContainerImageID(id string) string {
-	id = strings.Replace(id, "sha256:", "", -1)
```

Comment:
To be concise with the other images, we need this back in.

GCR and docker also include the "sha256" bit. Perhaps you can re-use `containerid.MondooContainerImageID`.